server/migrations/1616443559929_coordinate-sites-and-participants.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len, no-console */
+const { dbClient, collections } = require('../db');
+const { getPointsFromPostalCodes } = require('../services/geocodes.js');
+
+exports.up = async () => {
+  const participants = await dbClient.db[collections.PARTICIPANTS].findDoc(
+    { coords: undefined },
+  );
+
+  const sites = await dbClient.db[collections.EMPLOYER_SITES].findDoc(
+    { coords: undefined },
+  );
+
+  const postalCodes = participants.concat(sites).map((entry) => entry.postalCode);
+  const coords = await getPointsFromPostalCodes(postalCodes);
+
+  const promises = participants.map((participant) => dbClient.db[collections.PARTICIPANTS].updateDoc(
+    { id: participant.id },
+    { coords: coords[participant.postalCode] || { match: null } },
```

Comment:
I'm just trying to understand how the potential space in the postal codes is being handled here - I think we would be handling it on line 15 with the call to getPointsFromPostalCodes but then here we're doing an exact lookup?

Reply:
I think I might have missed handling it here, good catch. None of the entries in the coords object have spaces, but the postal code attached to participant can.